packages/block-editor/src/components/use-on-block-drop/index.js

```diff
@@ -255,17 +254,11 @@ export default function useOnBlockDrop(
 			if ( ! Array.isArray( blocks ) ) {
 				blocks = [ blocks ];
 			}
-
 			const clientIds = getBlockOrder( targetRootClientId );
 			const clientId = clientIds[ targetBlockIndex ];
-			const blocksClientIds = blocks.map( ( block ) => block.clientId );
-			const areGroupableBlocks = isGroupable( [
-				...blocksClientIds,
-				clientId,
-			] );
 			if ( operation === 'replace' ) {
 				replaceBlocks( clientId, blocks, undefined, initialPosition );
-			} else if ( operation === 'group' && areGroupableBlocks ) {
```

Comment:
At this point, it is guaranteed that at least one of the Row block or Gallery block can be created, so `areGroupableBlocks`  is not necessary.

Reply:
Just double-checking, is that because this check is now happening in the `throttle`'d function instead, prior to displaying the drop indicator and setting the drop target?

It looks like that other code also calls `isGroupable`, too 👍. From memory the other thing we needed to check was that the hovered over block can be removed, so as long as the check is happening _somewhere_ that sounds good to me!

Reply:
> Just double-checking, is that because this check is now happening in the "throttle"'d function instead, prior to displaying the drop indicator and setting the drop target?

Yes. If the `areGroupableBlocks` check is present here, the gallery block will not be created when the following conditions are met:

- The Group block is unregistered
- he dragged blocks and the target block are all images

Reply:
Thanks for clarifying!